evennia/utils/tests/test_containers.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+
+from evennia.utils import containers
+from django.conf import settings
+from evennia.utils.utils import class_from_module
+
+_BASE_SCRIPT_TYPECLASS = class_from_module(settings.BASE_SCRIPT_TYPECLASS)
+
+class GoodScript(_BASE_SCRIPT_TYPECLASS):
+  pass
+
+class BadScript:
+  """Not subclass of _BASE_SCRIPT_TYPECLASS,"""
+  pass
+
+class WorseScript(_BASE_SCRIPT_TYPECLASS):
+  """objects will fail upon call"""
+  @property
+  def objects(self):
+    from evennia import module_that_doesnt_exist
+
+class TestGlobalScriptContainer(unittest.TestCase):
+
+  def setUp(self):
+    settings.GLOBAL_SCRIPTS = {}
```

Comment:
This should not be changed on the `settings` object directly. This could affect other test-runs. Instead you use `override_settings` or `mock.patch` to replace this in-place for this test only. 

If you use an approach like this, you should restore the settings afterwards, by saving their real value and then putting them back in a `tearDown` method (but this could cause issues when running tests in parallel).